core/merge.js

```diff
@@ -156,6 +156,11 @@ class Merge {
     } else if (file.sides.local && file.sides.local >= file.sides.remote) {
       // The file was updated on local after being synched to remote
       return this.updateFileAsync(side, doc)
+    } else if (file.sides.local && file.sides.local < file.sides.remote) {
+      delete doc.remote
+      delete doc.sides
+      metadata.markSide('local', doc)
+      return this.resolveConflictAsync('local', doc, doc)
```

Comment:
Maybe, instead of adding a new `else if` in this function, we can modify https://github.com/cozy-labs/cozy-desktop/pull/1366/files#diff-6d18f9e7a792437c06b6f95186c97b88R176 to always resolve the conflict on local. It won't change anything from your PR if `file.sides.local` exist. If it doesn't exist, it looks safe to me to resolve the conflict on local (and it's probably better).

What do you think?

Reply:
Yes, I mean the last branch where we resolve the conflict remotely.